packages/frontend-2/assets/css/tailwind.css

```diff
@@ -51,15 +52,18 @@
   .caption {
     @apply text-xs;
   }
-}
 
-/**
- * Ensuring ring offset color always defaults to bg color
- */
-*,
-[type='checkbox']:focus,
-[type='radio']:focus {
-  @apply ring-offset-background;
+  .default-width {
+    @apply mx-auto max-w-7xl px-4 sm:px-6 lg:px-8;
+  }
+
+  .background-special {
+    @apply bg-[radial-gradient(ellipse_at_top_left,_var(--tw-gradient-stops))] from-background-primary to-primary dark:bg-blue-700 dark:from-sky-400 dark:via-blue-700 dark:to-indigo-900;
+  }
+
+  .link {
+    @apply text-blue-500 dark:text-blue-600;
```

Comment:
Make sure TextLink.vue is adjusted with these styles

Reply:
removed, it was a superfluous class